manager/ui/war/plugins/api-manager/ts/modals.ts

```diff
@@ -59,7 +59,8 @@ module ApimanModals {
         function ($location, $rootScope, $scope, $uibModalInstance, OrgSvcs, Configuration, PageLifecycle, client, params) {
         
         $scope.confirmClientName = '';
-        
+        $scope.client = client;

```

Comment:
Just for understanding what does these line do:

```javascript
$scope.client = client;
$scope.org = org
$scope.plan = plan;
```

Reply:
Hi @volkflo 
I've added this because of it not accessible from a template with {{ client.name }} or {{ org.name }} in placeholders of deletion modal.

When modal was show the textbox placeholder was empty. These lines are the fix ;)